src/index.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
 import * as serviceWorkerRegistration from './serviceWorkerRegistration';
 import reportWebVitals from './reportWebVitals';
+import BotNav from './Componants/navbar';
+
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BotNav />
```

Comment:
Let's keep the `App` component and put the `<BotNav />` in there after the `<header/>` closing tag. We'll be dropping in more components in the main `App` so let's set up that pattern now.